tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -5258,3 +5260,18 @@ TEST_F(CompilerTest, ViewID) {
   CodeGenTestCheck(L"..\\CodeGenHLSL\\viewid\\viewid17.hlsl");
   CodeGenTestCheck(L"..\\CodeGenHLSL\\viewid\\viewid18.hlsl");
 }
+
+TEST_F(CompilerTest, ShaderCompatSuite) {
+  using namespace std::experimental::filesystem;
```

Comment:
We should use the llvm::sys::fs for this instead of std::experimental::filesystem. Looks like the llvm library should provide everything we need.